google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -324,6 +324,7 @@ public Bucket create(Bucket bucket, Map<Option, ?> options) {
           .setProjection(DEFAULT_PROJECTION)
           .setPredefinedAcl(Option.PREDEFINED_ACL.getString(options))
           .setPredefinedDefaultObjectAcl(Option.PREDEFINED_DEFAULT_OBJECT_ACL.getString(options))
+          .setUserProject(Option.USER_PROJECT.getString(options))
```

Comment:
No way, the auditing caught that userProject wasn't set for this calls?

Reply:
Indeed it did.